modules/ui/intro/point.js

```diff
@@ -143,7 +143,7 @@ export function uiIntroPoint(context, reveal) {
             context.history().on('change.intro', deleted);
 
             setTimeout(function() {
-                var node = d3.select('.radial-menu-item-delete').node();
+                var node = d3.select('.edit-menu-item-delete').node();
```

Comment:
Reminder to 1. keep `radial-menu` classes around for backward compat.  2. test what this actually does when we run through the tutorial.